storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options that are not possible with in-tree volume plug-ins, such as volume snapshots.
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned to be migrated to the CSI driver at that time.
+====
+
+The AWS Elastic Block Store (EBS) CSI Operator provides the CSI driver needed for this extensibility in {product-title}.
+
+:FeatureName: AWS EBS CSI Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+For information about dynamically provisioning AWS EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-using-aws-ebs[Persistent Storage Using AWS Elastic Block Store].
+
+include::modules/persistent-storage-csi-ebs-operator-install.adoc[leveloffset=+1]
```

Comment:
Do we want to add a text explaining how the user can now create volumes using the StorageClass created by the operator? Or do you think that's not necessary?

Reply:
Yeah, I think it's useful to mention that the StorageClass is created by the Operator. I've added a sentence about this.

Reply:
Do you think we may describe sc can be creaed by Operator or created by customer if required some paramters.

Reply:
Great idea, @chao007. I've added a link to the section that describes defining the sc. (Unfortunately, I can't xref directly to the sc module-level.)